app/gimletlet/app-mgmt.toml

```diff
@@ -49,7 +49,7 @@ task-slots = ["sys", "user_leds"]
 
 [tasks.net]
 name = "task-net"
-stacksize = 3000
+stacksize = 8000
```

Comment:
I'm assuming this is an unrelated change which you discovered when building every `app-*.toml`?